test/absinthe/middleware/async_test.exs

```diff
@@ -75,12 +75,39 @@ defmodule Absinthe.Middleware.AsyncTest do
     assert {:ok, %{data: %{"asyncBareThing" => "bare task"}}} == Absinthe.run(doc, Schema)
   end
 
-  test "can resolve a field using the normal test helper" do
+  test "can resolve a field using the normal test helper and emit telemetry event", %{test: test} do
```

Comment:
I extended this existing test with extra telemetry. I'm happy to change to a separate test OR add telemetry assertions on the existing ones - when applicable.